README.md

```diff
@@ -45,6 +45,8 @@ To configure the Signal Sciences agent the following environment variables must
 
 `SIGSCI_SERVER_HOSTNAME` (optional)
 
+`SIGSCI_INTERNAL_AGENT_DOWNLOAD_URL` (optional)
```

Comment:
this can be `SIGSCI_AGENT_DOWNLOAD_URL` and we should explain that it should be a fully qualified http(s) path to a tar, gzipped agent file.

Reply:
updated the name and added a comment